src/or_trajopt/planning.py

```diff
@@ -560,11 +565,8 @@ def PlanToTSR(self, robot, tsrchains, is_interactive=False, **kw_args):
         with robot.CreateRobotStateSaver(p.ActiveDOF):
             # Set robot DOFs to DOFs in optimization problem
             prob.SetRobotActiveDOFs()
-
             # Check that trajectory is collision free
-            from trajoptpy.check_traj import traj_is_safe
-            if not traj_is_safe(waypoints, robot):
-                return PlanningError("Result was in collision.")
+            self._checkCollisionForIKSolutions(robot, waypoints)
```

Comment:
I'm not sure this is the correct logic here.  This only checks for collision immediately at each waypoint, but the trajectory could collide in between waypoints.

@mkoval: Does `prpy` have a utility function to check if an entire trajectory is collision free at the active DOF resolution?